src/components/Modals/ModalInfo.jsx

```diff
@@ -30,15 +30,16 @@ class ModalInfoWallet extends React.Component {
 					<div className="modal-body">
 						<div className="info-row">
 							<div className="info-title">Version:</div>
-							<div className="info-value">1.4.4 <a href="#" target="_blank"> <span className="icon-commit" />k8b3c4e</a></div>
+							{/* eslint-disable-next-line no-undef */}
+							<div className="info-value">{version} <a href="#" target="_blank"> <span className="icon-commit" />{COMMITHASH.substring(0, 7)}</a></div>
```

Comment:
one more thing
lets link would be clicable and forward us to 
`echo.org` and
`https://github.com/echoprotocol/echo-wallet/commit/${commitHash}`